.buildkite/Dockerfile

```diff
@@ -20,13 +20,17 @@ RUN mkdir -p /gcloud/ && \
 RUN pip install \
 	--no-cache-dir \
 	pipx && \
-	pipx install pdm~=2.4.0 && \
-	pipx install pdm~=2.3.0 --suffix 23 && \
-	pipx install pdm~=2.4.0 --suffix 24 && \
-	pipx install pdm~=2.5.0 --suffix 25 && \
+	pipx install --pip-args='--no-cache-dir' pdm~=2.4.0 && \
```

Comment:
Perhaps there's value in not having a default version and instead force users to be explicit about which version they want to use